monkey/infection_monkey/post_breach/actions/communicate_as_new_user.py

```diff
@@ -23,8 +23,8 @@
 
 class CommunicateAsNewUser(PBA):
     """
-    This PBA creates a new user, and then pings google as that user. This is used for a Zero Trust test of the People
-    pillar. See the relevant telemetry processing to see what findings are created.
+    This PBA creates a new user, and then creates HTTP/S requests as that user. This is used for a Zero Trust test of the
```

Comment:
S'not HTTP, only HTTPS since you hardcode....

Reply:
Fixed